jest-setup.ts

```diff
@@ -71,3 +72,5 @@ jest.mock( './src/hooks/use-ai-icon', () => ( {
 } ) );
 
 global.ResizeObserver = require( 'resize-observer-polyfill' );
+
+jestPreviewConfigure( { autoPreview: true } );
```

Comment:
Should we avoid setting this on CI?
I'm not sure if there's a direct side effect on having this configuration, but I also don't see any reason why we would benefit enabling it in CI.